pkg/driver/nri_hooks.go

```diff
@@ -57,29 +57,32 @@ func (np *NetworkDriver) Synchronize(_ context.Context, pods []*api.PodSandbox,
 	return nil, nil
 }
 
+var createContainerImpl = (*NetworkDriver).createContainer
```

Comment:
Do we really need these mocks? Can't we manage without them? If our answer is yes, I'd wonder if we even need the unit tests for the metrics, because if we've faked this function, then we're essentially testing whether an "if/else" is working correctly (which would always be the case)

Reply:
To test the label, there would have to be some modification required, yes. I agree with you, we can remove these tests.